core/invoker/src/main/scala/whisk/core/containerpool/ContainerPool.scala

```diff
@@ -192,13 +222,62 @@ class ContainerPool(childFactory: ActorRefFactory => ActorRef,
     val ref = childFactory(context)
     val data = NoData()
     freePool = freePool + (ref -> data)
+//    if (active && replicatePrewarm) {
+//      replicateFreePool(data)
+//    }
+
     ref -> data
   }
 
   /** Creates a new prewarmed container */
   def prewarmContainer(exec: CodeExec[_], memoryLimit: ByteSize) =
     childFactory(context) ! Start(exec, memoryLimit)
 
+//  def attachPrewarmContainer(id: ContainerId, ip: ContainerAddress, exec: CodeExec[_], memoryLimit: ByteSize) =
+//    childFactory(context) ! Attach(id, ip, exec, memoryLimit)
+//  def attachFreeContainer(id: ContainerId,
+//                          ip: ContainerAddress,
+//                          invocationNamespace: EntityName,
+//                          action: ExecutableWhiskAction) =
+//    childFactory(context) ! AttachFree(id, ip, invocationNamespace, action)
+//
+//  def replicateFreePool(cd: ContainerData) =
+//    cd match {
+//      case data: WarmedData =>
+//        data.action.fullyQualifiedName(true).toString
+//        data.action.rev.rev
+//        replicator ! Update(freePoolKey, ORSet.empty[ReplicatedWarm], WriteLocal)(
+//          _ + ReplicatedWarm(
+//            data.container.id,
+//            data.container.addr,
+//            data.lastUsed,
+//            data.invocationNamespace.toString,
+//            FullyQualifiedEntityName.serdes.write(data.action.fullyQualifiedName(true)),
+//            data.action.rev.asString))
+//      case _ => logging.info(this, s"skipping freepool replication of ${cd}")
+//    }
+//
+//  def unreplicateFreePool(cd: ContainerData) =
+//    cd match {
+//      case data: WarmedData =>
+//        replicator ! Update(freePoolKey, ORSet.empty[ReplicatedWarm], WriteLocal)(
+//          _ - ReplicatedWarm(
+//            data.container.id,
+//            data.container.addr,
+//            data.lastUsed,
+//            data.invocationNamespace.toString,
+//            FullyQualifiedEntityName.serdes.write(data.action.fullyQualifiedName(true)),
+//            data.action.rev.asString))
+//      case _ => logging.info(this, s"skipping freepool unreplication of ${cd}")
+//    }
+//
+//  def replicatePrewarmPool(data: PreWarmedData) =
+//    replicator ! Update(prewarmPoolKey, ORSet.empty[ReplicatedPrewarm], WriteLocal)(
+//      _ + ReplicatedPrewarm(data.container.id, data.container.addr, data.kind, data.memoryLimit))
+//  def unreplicatePrewarmPool(data: PreWarmedData) =
+//    replicator ! Update(prewarmPoolKey, ORSet.empty[ReplicatedPrewarm], WriteLocal)(
+//      _ - ReplicatedPrewarm(data.container.id, data.container.addr, data.kind, data.memoryLimit))
+
```

Comment:
Is it necessary to include large blocks of commented out code in this PR?  Seems a bit ugly (same thought applies to the smaller hunks related to pool replication above and below).

Reply:
no it is not necessary; I meant to clean these up...